src/osemgrep/language_server/scan_helpers/Scan_helpers.ml

```diff
@@ -263,6 +263,18 @@ let refresh_rules server =
   Lwt.async (fun () ->
       let%lwt () = Session.cache_session server.session in
       end_progress token;
+      let refresh_rules_notif =
```

Comment:
probably best to isolate this notification creation stuff to a `RPC_server` helper